api/src/main/java/edu/cornell/mannlib/vitro/webapp/auth/policy/PolicyLoader.java

```diff
@@ -478,9 +499,14 @@ protected void processQuerySolution(QuerySolution qs) {
         return values;
     }
 
-    public String getEntityValueSetUri(AccessOperation ao, AccessObjectType aot, String role) {
-        long expectedSize = 3;
-        String queryText = getDataSetByKeyQuery(new String[] { }, new String[] { ao.toString(), aot.toString(), role });
+    public String getEntityValueSetUri(AccessOperation ao, AccessObjectType aot, String role,
+            String... namedKeyComponents) {
+        int expectedSize = 3 + namedKeyComponents.length;
+        String[] ids = Arrays.copyOf(namedKeyComponents, expectedSize);
```

Comment:
Can you please align this with https://github.com/vivo-project/Vitro/pull/433/files#diff-2ccd8e24d26e6788b9443cf88e40fd036d2e231a5bc70baa98ebfb756ef0b7f3R101?